src/main/Member.js

```diff
@@ -423,14 +423,41 @@ export default class Member {
     }
 
     /**
-     * Triggers a step up notification on the user's app
+     * Triggers a token step up notification on the user's app
      *
-     * @param {object} stepUp - can be tokenStepUp or requestStepUp
+     * @param {String} tokenId - token id
      * @return {Promise} - notification status
      */
-    triggerStepUpNotification(stepUp) {
```

Comment:
is this being used anywhere?

Reply:
Yeah this is used when you need to step up for access and transfer tokens.

Reply:
yes web-app uses it
https://github.com/tokenio/web-app/blob/master/src/actions/utils/commonUtils.js#L150

Reply:
OK as long as the backend calls are not removed that's fine. we need to make sure there is no breaking change here, and we can just use the new call when we update the sdk in web-app

Reply:
Actually access tokens might not use it now that I think about it, but it's definitely used for transfer tokens.

Reply:
Mmmh, this would be a breaking change. Maybe keep the legacy triggerStepUpNotification method for tokens (since it's being used now) and add triggerBalance/triggerTransaction?

Reply:
sure, is there any difference between the new one and the old one? besides adding token to the name

Reply:
None. I'll revert it.